packages/package-lock.json

```diff
@@ -0,0 +1,848 @@
+{
```

Comment:
Is this file suppose to be here? I see a copy of this already in the client folder.